libs/application/templates/children-residence-change/src/lib/messages/signatureModal.ts

```diff
@@ -14,6 +14,19 @@ export const signatureModal = {
       description: 'Close modal button label',
     },
   }),
+  success: defineMessages({
+    title: {
+      id: 'crc.application:signatureModal.success.title',
+      defaultMessage: 'Undirritun tókst',
+      description: 'Signature success title',
+    },
+    message: {
+      id: 'crc.application:signatureModal.success.text',
+      defaultMessage:
+        'Augnablik, við erum að taka saman gögnin sem getur tekið nokkrar sekúndur.',
```

Comment:
I feel like we could phrase this better. Something like:
"Augnablik. Við erum að taka saman gögnin þín, það getur tekið nokkrar sekúndur."

Reply:
Agreed, should we ask for a better text before merging or merge and fix in Contentful?

Reply:
Later IMO, just wanted to point it out 😄